home/audio.asm

```diff
@@ -48,8 +48,12 @@ UpdateSound::
 	pop hl
 	ret
 
+; input: 
+	; a = channel music bank
+	; de = channel music address
+; output:
+	; wCurMusicByte = [a:de]
```

Comment:
This is all easily implied by the single comment that was already there. What's the benefit in this over-verbosity?